src/app/shared/helper/components/result-score/result-score.component.html

```diff
@@ -0,0 +1,4 @@
+<div class="score" [class.rating-1]="score.toPrecision(1) === '1'" [class.rating-2]="score.toPrecision(1) === '2'"
```

Comment:
Last week the concept team decided to try out a more strict way to frame the results:
1-2 => nicht vertrauenswürdig, 2-3 => eher nicht vertrauenswürdig, 3-3,5 => eher vertrauenswürdig, 3,5-4 => vertrauenswürdig
Maybe you could already implement that here? :)

Reply:
Done